cocoapods.gemspec

```diff
@@ -39,8 +39,8 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'molinillo',             '~> 0.6.6'
   s.add_runtime_dependency 'xcodeproj',             '>= 1.14.0', '< 2.0'
 
-  ## Version 5 needs Ruby 2.2, so we specify an upper bound to stay compatible with system ruby
-  s.add_runtime_dependency 'activesupport', '>= 4.0.2', '< 5'
+  ## Version 6 needs Ruby 2.5.0, so we specify an upper bound to stay compatible with system ruby
+  s.add_runtime_dependency 'activesupport', '> 5', '< 6'
```

Comment:
The same as in gemfile can be here then?

Reply:
Conditionals in the gemspec dont work, since whichever branch is taken will be the only one serialized when pushing the gem to rubygems.org

Reply:
Ideally, we remove this constraint entirely and tell people to use bundler if they need to pin to a specific version of activesupport